portfolio/src/main/webapp/about.js

```diff
@@ -1,82 +1,106 @@
-function loadInFact(index, fact_index, isTruth) {
-    let fact = isTruth ? truths[fact_index] : lies[fact_index];
+let correctAnswer = -1;
 
-    switch(index) {
-        case 0:
-            document.getElementById("first_txt").innerHTML = fact[0];
-            document.getElementById("first_img").src = fact[1];
-            break;
-        case 1:
-            document.getElementById("second_txt").innerHTML = fact[0];
-            document.getElementById("second_img").src = fact[1];
-            break;
-        case 2:
-            document.getElementById("third_txt").innerHTML = fact[0];
-            document.getElementById("third_img").src = fact[1];
-            break;
-    }
+/**
+ * Advances the game to the next question. Loads in the first question 
+ * game has not yet started. Otherwise, checks answer against user 
+ * selected anwer and alerts if a lie or truth has been chosen.
+ */
+function next()  {
+    let gameStatus = document.getElementById('game-status');
+    if (gameStatus.innerHTML=='Press Start to Begin!') {
+        gameStatus.innerHTML = 1;
+        document.getElementById('advance-game-btn').innerHTML = 'Next';
+    } else {
+        let chosenAnswer = -1;
+        const chosenRadios = document.querySelectorAll('input[name="selected-answer"]');
+        for (chosenRadio of chosenRadios) {
+            if(chosenRadio.checked) {
+                chosenAnswer = chosenRadio.value;
+            }
+        }
+        if (chosenAnswer != -1) {
+            if (correctAnswer == chosenAnswer) {
+                alert('This is the lie!');
+            } else {
+                alert('This is the truth!');
+            }
+        }
+    } 
+    const truthIndices = loadInTruth();
+    correctAnswer = loadInLie(truthIndices);
 }
 
+/**
+ * Populates the page with two true facts in randomized indexes
+ * 
+ *@return {Array} an array populated with two numbers containing
```

Comment:
Insert a space between the asterisk and the at sign.

Reply:
Added space